tree-sitter.json

```diff
@@ -8,7 +8,8 @@
       "file-types": [
         "cr"
       ],
-      "injection-regex": "crystal"
+      "injection-regex": "crystal",
+      "highlights": "queries/nvim/highlights.scm"
```

Comment:
Is it possible to set multiple highlight files to be tested?

Reply:
I don't think so